crates/nargo/tests/test_data/2_div/src/foo.nr

```diff
@@ -0,0 +1,3 @@
+fn hello(x : pub Field) -> Field {
```

Comment:
This file is not being used, so it should be removed. Can you do it for all of the examples?

Reply:
yes, it's done.

Reply:
Yes, it's done.